src/main/java/com/blamejared/crafttweaker/impl_native/world/ExpandWorld.java

```diff
@@ -1,125 +1,384 @@
 package com.blamejared.crafttweaker.impl_native.world;
 
+import javax.annotation.Nullable;
+
+import org.openzen.zencode.java.ZenCodeType;
+
 import com.blamejared.crafttweaker.api.annotations.ZenRegister;
 import com.blamejared.crafttweaker.api.data.IData;
 import com.blamejared.crafttweaker.api.data.NBTConverter;
 import com.blamejared.crafttweaker.impl.data.MapData;
 import com.blamejared.crafttweaker.impl.util.MCDirection;
 import com.blamejared.crafttweaker_annotations.annotations.Document;
 import com.blamejared.crafttweaker_annotations.annotations.NativeTypeRegistration;
+
 import net.minecraft.block.BlockState;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.nbt.CompoundNBT;
 import net.minecraft.tileentity.TileEntity;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.World;
 import net.minecraft.world.biome.Biome;
-import org.openzen.zencode.java.ZenCodeType;
+import net.minecraft.world.server.ServerWorld;
 
+/**
+ * Worlds represent a dimension within the game. They are used to interact with 
+ * the blocks, mobs, and other variables within that dimension.
+ * 
+ * @docParam this world
+ */
 @ZenRegister
 @Document("vanilla/api/world/MCWorld")
 @NativeTypeRegistration(value = World.class, zenCodeName = "crafttweaker.api.world.MCWorld")
 public class ExpandWorld {
     
+    @ZenCodeType.Method
+    @ZenCodeType.Caster(implicit = false)
+    public static ServerWorld asServerWorld(World internal) {
+        return (ServerWorld) internal;
+    }
+    
+    /**
+     * Checks if the world is remote. This is always true on the rendering
+     * thread.
```

Comment:
I feel like the comment about the rendering thread would be confusing to people who don't know how the threads work

Reply:
I used this language because I wanted to avoid client/server terminology. I am not sure how you typically explain this to people.